core/net/ipv6/uip-nd6.c

```diff
@@ -518,7 +518,7 @@ na_input(void)
         goto discard;
       }
 
-#if NETSTACK_RADIO != ble_mode
+#if NETSTACK_RADIO != ble_mode_driver
```

Comment:
> I don't think the C preprocessor can see symbolic names; so, I suspect that check cannot be done that way. For an example, try compiling this code -- it will print "equals":
```C
int abc, def;
#define ABC abc
#if ABC == def
#warning equals
#endif
```
`#if` sees macro names.  And, arithmetic operators see only numbers.  `abc` and `def` aren't #defined; therefore, they're "expanded" into zeroes.
```C
#if 0 == 0
```